package/build.sh

```diff
@@ -19,20 +19,18 @@ mkdir -p package/log
 if [ ! -f "package/cache/xen-intermediate-$IMAGE-$XEN_HASH.tar.gz" ]
 then
     echo Building Xen intermediate $XEN_HASH...
-    DOCKER_CMD="docker build --build-arg 'IMAGE=$IMAGE' -f package/Dockerfile-xen -t xen-intermediate ."
 
-    if [ ! -z "$TRAVIS_JOB_ID" ]; then
-        # suppress Xen build logs when we are in CI
-        sh -c "$DOCKER_CMD 2>&1 >package/log/xen-build.log"
-    else
-        sh -c "$DOCKER_CMD"
-    fi
```

Comment:
Are you sure about this? Xen build generates so enormous output on stdout/stderr that Travis aborts the job due to exceeded log size limit

Reply:
ah, but I've forgot we don't use travis anymore :trollface:

Reply:
Yeap, on GA it doesn't seem to matter.